packages/@react-spectrum/overlays/src/Underlay.tsx

```diff
@@ -15,11 +15,12 @@ import React from 'react';
 import underlayStyles from '@adobe/spectrum-css-temp/components/underlay/vars.css';
 
 interface UnderlayProps {
-  isOpen?: boolean
+  isOpen?: boolean,
+  transparent?: boolean
```

Comment:
perhaps `isTransparent` instead to conform with the [guidelines](https://github.com/adobe/react-spectrum/blob/main/specs/api/Guidelines.md#boolean-props)